### DISCUSSION TITLE
Confusion on How to Use in Blade Views and AJAX Calls

### DISCUSSION THREAD

**Discussion by SpencerCloud** [spatie/laravel-ray#127] [Q&A] [answered]
Hey Spatie,

I am enjoying using Ray out so far, but one thing that I haven't figured out yet is how to use it with Blade Views and AJAX calls.

I am using Ray with Laravel and whenever I call Ray using the [special Blade syntax](https://spatie.be/docs/ray/v1/usage/laravel#using-ray-in-blade-views) or try it in a normal PHP file called from an AJAX request, nothing happens.

It's possible that I'm missing something key, but I have looked through the documentation and there doesn't seem to be anything that references different configurations for these use cases, or what to do for troubleshooting.

I am using Laradock just in case that is relevant, and developing on macOS Big Sur 11.1.

Thanks!

**Comment by patinthehat** [comment, marked as answer]
Hey @SpencerCloud, hopefully one of these will point you in the right direction:
- Is the `enable` configuration setting set to false? _(unless you changed it, the default value should be fine)_
- Did you set `RAY_ENABLED=false` in your `.env`? _(if undefined, it defaults to `true`)_
- Does the `port` configuration setting match the same setting in the Ray app (Menu -> Preferences)? _(the default is `23517`)_
- Since you're using Laradock, did you follow the instructions [for docker](https://spatie.be/docs/ray/v1/environment-specific-configuration/docker)?

**Reply by SpencerCloud** [reply]
Thanks! Most of those configurations were right, but changing my `RAY_HOST` from "localhost" to "host.docker.internal" seems to have done the trick.
